rest-client/src/main/java/com/gentics/mesh/rest/client/AbstractMeshRestHttpClient.java

```diff
@@ -28,6 +27,8 @@
 
 	protected boolean disableAnonymousAccess = false;
 
+	private String baseUri;
```

Comment:
Keep the default value of "/api/v1/". Don't handle the default value in the getter.